Makefile.sp

```diff
@@ -114,7 +115,24 @@ sp-vale: sp-install
 	@echo ""
 	@. $(VENV); vale --config "$(SPHINXDIR)/vale.ini" --glob='*.{md,txt,rst}' $(TARGET)
 
-
+sp-pdf-prep: sp-install
+	@for packageName in $(REQPDFPACKS); do (dpkg-query -W -f='$${Status}' $$packageName 2>/dev/null | \
+        grep -c "ok installed" >/dev/null && echo "Package $$packageName is installed") || \
+        (echo "PDF generation requires the installation of the following packages: $(REQPDFPACKS)" && \
+        echo "" && echo "sudo make pdf-prep-force will install these packages" && echo "" && echo \
+        "Please be aware these packages will be installed to your system" && false) ; done
+
+sp-pdf-prep-force:
+	apt-get update
+	apt-get upgrade -y
+	apt-get install --no-install-recommends -y $(REQPDFPACKS) \
+
+sp-pdf: sp-pdf-prep
+	@. $(VENV); sphinx-build -M latexpdf "$(SOURCEDIR)" "_build" $(SPHINXOPTS)
+	@. $(VENV); rm ./_build/latex/front-page-light.pdf || true
+	@. $(VENV); rm ./_build/latex/normal-page-footer.pdf || true
+	@. $(VENV); find ./_build/latex -name "*.pdf" -exec mv -t ./ {} +
+	@. $(VENV); rm -r _build
```

Comment:
Wait - you can't do that! `_build` also has the HTML build, so if I build HTML first and then PDF, my HTML build is gone.

Also, we need a way to add the PDF to `.gitignore` so it's not accidentally checked in. So my suggestion is to use some temp directory for building, and then move the PDF to the `$(BUILDDIR)` folder (which is automatically excluded).

Reply:
I might move the PDF to the `_build` directory and delete the `_build/latex` directory. Which should work, as `_build` is already ignored.

Reply:
Fixed as I described with [9e0a32e](https://github.com/canonical/sphinx-docs-starter-pack/pull/264/commits/9e0a32ecedc9022d4174d8d57a3308820b5b8f2e)